 0.0% done, time left: unknown                               100.0% done, time left: unknown  finished
The dynamic slice for criterion [org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68:*]:
org.apache.commons.lang3.exception.CloneFailedException.<init>:39 ALOAD 0
org.apache.commons.lang3.exception.CloneFailedException.<init>:39 ALOAD 1
org.apache.commons.lang3.exception.CloneFailedException.<init>:39 INVOKESPECIAL java/lang/RuntimeException.<init>(Ljava/lang/String;)V
org.apache.commons.lang3.exception.CloneFailedExceptionTest.main:33 INVOKEVIRTUAL org/apache/commons/lang3/exception/CloneFailedExceptionTest.testWithoutCause()V
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:66 NEW org/apache/commons/lang3/exception/CloneFailedException
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:66 DUP
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:66 LDC "Exception message"
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:66 INVOKESPECIAL org/apache/commons/lang3/exception/CloneFailedException.<init>(Ljava/lang/String;)V
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:66 ASTORE 1
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68 LDC "Wrong exception message"
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68 LDC "Exception message"
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68 ALOAD 1
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68 INVOKEVIRTUAL java/lang/Exception.getMessage()Ljava/lang/String;
org.apache.commons.lang3.exception.CloneFailedExceptionTest.testWithoutCause:68 INVOKESTATIC org/junit/Assert.assertEquals(Ljava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V

Slice consists of 26 bytecode instructions.
Computation took 0.52 seconds.
